app/domain/logs.rb

```diff
@@ -133,6 +133,17 @@ module AuthnK8s
 
       module AuthnAzure
 
+        # TODO: get security approval for this message
+        ExtractedXmsMiridFromAzureToked = ::Util::TrackableLogMessageClass.new(
+          msg: "Extracted xms_mirid '{0}' from Token",
```

Comment:
"Token" here and "token" in the next log message are not inconsistent. Let's be consistent in terms of T or t when referencing "token"